lib/route_model.dart

```diff
@@ -0,0 +1,55 @@
+import 'dart:core';
+import 'dart:math';
+
+import 'package:sail_routing_dart/cart_point.dart';
+
+class RouteModel {
+  Cart_Point _start;
+  Cart_Point _end;
+  List<Cart_Point> _intermediate_points;
```

Comment:
https://www.dartlang.org/guides/language/effective-dart/style I think lowerCamelCase for _intermetiate_points